.semaphore/paprika-status-a11y.yml

```diff
@@ -21,7 +21,7 @@ blocks:
       jobs:
         - name: "Start paprika-status script"
           commands:
-            - nvm use
+            - nvm install
```

Comment:
Oh! So this will _install_ the node version in `.nvmrc`?

Reply:
yes

Reply:
TIL 😄